packages/browser/docs/api/_templates/docs-navbar.html

```diff
@@ -12,16 +12,18 @@
     <div id="navbar-menu" class="col-lg-9 collapse navbar-collapse">
       <ul id="navbar-main-elements" class="navbar-nav mr-auto">
           <li class="nav-item">
-             <a class="nav-link" href="{{theme_ess_docs}}"> Enterprise Solid Server </a>
+             <a class="nav-link" href="{{theme_ess_docs}}">Enterprise Solid Server</a>
           </li>
-          <li class="nav-item"><a class="nav-link" href="#">Client Libraries </a>
+          <li class="nav-item"><a class="nav-link" href="#">Developer Tools</a>
                <ul class="nav-children">
                      <li class="nav-item nav-childitem">
-                        <a href="{{theme_clientlibjs_docs}}"> JavaScript</a>
+                        <a href="{{theme_clientlibjs_docs}}">JavaScript Client Libraries</a>
+                     </li>
+                     <li class="nav-item nav-childitem">
+                        <a href="{{theme_reactsdk_docs}}">JavaScript React SDK</a>
```

Comment:
@kay-kim Given that https://github.com/inrupt/docs-client-lib-js/pull/9 says "Wait until next week", should this wait as well?